src/components/tl/cuda/tl_cuda_cache.c

```diff
@@ -145,6 +145,9 @@ ucc_tl_cuda_map_memhandle(const void *d_ptr, size_t size,
                           cudaIpcMemHandle_t mem_handle, void **mapped_addr,
                           ucc_tl_cuda_cache_t *cache)
 {
+    if (d_ptr == NULL || size == 0) {
```

Comment:
should we set mapped_addr to some value, e.g. NULL? otherwise we return UCC_OK and leave *mapped_addr uninitialized which looks fishy

Reply:
set `*mapped_addr` to NULL.